src/lstm.js

```diff
@@ -47,12 +47,17 @@ export function lstm(input, weight, recurrentWeight, steps, hiddenSize,
         initialCellState, new Array(sizeOfShape(initialCellState)).fill(0));
   }
 
-  let sequence;
   const currentWeight = [];
   const currentRecurrentWeight = [];
   const currentBias = [];
   const currentRecurrentBias = [];
   const currentPeepholeWeight = [];
+  let forwardSequence = null;
+  let backwardSequence = null;
+  let currentHidden;
+  let currentCell;
+  let outputHidden;
```

Comment:
Be consistent with `forwardSequence`/`backwardSequence`?
```suggestion
  let outputHidden = null;
```